internal/cmd/commands/authenticate/password.go

```diff
@@ -67,12 +67,32 @@ func (c *PasswordCommand) Flags() *base.FlagSets {
 		Usage:  "The password associated with the login name. If blank, the command will prompt for the password to be entered interactively in a non-echoing way. Otherwise, this can refer to a file on disk (file://) from which a password will be read or an env var (env://) from which the password will be read.",
 	})
 
-	f.StringVar(&base.StringVar{
-		Name:   "auth-method-id",
-		EnvVar: "BOUNDARY_AUTH_METHOD_ID",
-		Target: &c.FlagAuthMethodId,
-		Usage:  "The auth-method resource to use for the operation.",
-	})
```

Comment:
@jimlambrt @johanbrandhorst (FYI @jefferai ) Ideas on this issue I ran into starting on line 76?